src/actions/Button.scss

```diff
@@ -135,13 +135,26 @@
     }
   }
 
-  &[data-variant="borderless-text"] {
+  &[data-variant="text"] {
     background-color: transparent;
-    border-color: transparent;
+    border: none;
+    border-radius: 0;
+    padding: 0;
     color: var(--seeds-color-primary);
     font-weight: normal;
     text-decoration: underline;
     gap: var(--seeds-s1);
+
+    &[data-size="sm"] {
+      padding: 0;
+      border-radius: 0;
```

Comment:
These are the same values as above is this needed anymore? Same for the data-size lg below

Reply:
Good point, some of the default vs. "md" size styles were hard to parse out…have made this clearer throughout.